scripts/build-ios.sh

```diff
@@ -81,6 +81,9 @@ for platform in "${PLATFORMS[@]}"; do
     esac
 done
 
+mkdir -p "$PROJECT_ROOT"/prebuilds
+test -e "$PROJECT_ROOT"/prebuilds/realm-js-ios.xcframework && cp -a "$PROJECT_ROOT"/prebuilds/realm-js-ios.xcframework
```

Comment:
Same with Android, I would appreciate if we didn't use the `prebuilds` directory to store the ios xcframework artifact.

Reply:
Also, why are we doing this in the first place? Won't this simply get overwritten in the subsequent steps of this script?